src/security/srv_acl.c

```diff
@@ -115,10 +115,7 @@ auth_token_dup(Auth__Token *orig)
 	uint8_t			*packed;
 	size_t			len;
 
-	/*
-	 * The most straightforward way to copy a protobuf struct is to pack
-	 * and unpack it.
-	 */
+	/* The most straightforward way to copy a protobuf struct is to pack and unpack it. */
```

Comment:
I'm not sure what happened here, I think at one point I'd modified the line of code immediately following this and re-flowed the comment so it would fit on one line rather than four but I must have backed out that part of the change and not spotted the comment change.  Have reverted.